bindings/core/tests/secrets_manager.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_sdk::client::{api::GetAddressesBuilderOptions, generate_mnemonic, secret::SecretManager};
+use iota_sdk_bindings_core::{call_secret_manager_method, Response, Result, SecretManagerMethod};
+
+#[tokio::test]
+async fn generate_addresses() -> Result<()> {
+    let mut secret_manager = SecretManager::try_from_mnemonic(&generate_mnemonic()?)?;
+
+    let method = SecretManagerMethod::GenerateAddresses {
+        options: GetAddressesBuilderOptions::default(),
+    };
+
+    let response = call_secret_manager_method(&mut secret_manager, method).await;
+    match response {
+        Response::GeneratedAddresses(addresses) => println!("{addresses:?}"),
+        _ => panic!("Unexpected response type"),
+    };
```

Comment:
Would be better to test a mnemonic/address pair we know is correct

Reply:
Updated

Reply:
I don't see it

Reply:
Ah sorry, replaced only in another test
Also updated this one now